### ISSUE TITLE
Function names with subscripts

### ISSUE THREAD

**Issue by adomasbaliuka** [korsbo/Latexify.jl#194]
The way subscripts are handled seems inconsistent between "functions" and "variables".

For example, 
```
julia> @latexrun f_a = π
L"$f_{a} = \pi$"
```
represents it in LaTeX as a subscripted variable. On the other hand, defining a function with the same syntax

```julia
julia> @latexrun f_b(x) = 2 * x
L"$\mathrm{f\_b}\left( x \right) = 2 \cdot x$"
```
escapes the underscore, such that it remains an underscore in the LaTeX representation. Is this difference between variables and functions on purpose?

**Comment by gustaphe** [comment]
Yes, there is some intent to this. Because Julia style guides encourage `snake_case_for_multiword_functions()` it's difficult to tell automatically whether an underscore is a subscript or a literal underscore.

When the overhaul (#123) is finished, this should be configurable. In the meanwhile you can use subscript characters (if they are defined in utf8).

See #103

**Comment by adomasbaliuka** [comment]
> In the meanwhile you can use subscript characters (if they are defined in utf8).

That doesn't always produce subscripts either...
```julia
julia> @latexify τ₁ # so far so good
L"$\tau{_1}$"
julia> @latexify τₙ # here, the underscore is escaped again...
L"$\tau\_n$"
```

By the way, thanks a lot for your great work! Not long ago I didn't expect to ever see connections between programming with published equations to an extent as Julia and this library might enable soon... Is there anything I can do to get these things working more closely to how I'm expecting it to, if that is in the interest of this project?

**Comment by gustaphe** [comment]
Disclaimer: I'm not a maintainer, just someone who follows updates on this package because it's amazing.

There is https://github.com/korsbo/Latexify.jl/pull/167 which I think solves this (and also contains @korsbo's thoughts on the default behaviour), sorry I forgot about that in my last reply. I don't know exactly where that PR stands right now.

**Comment by gustaphe** [comment]
This was closed with #215 : You can use `@latexify f_b(x) = x^2 snakecase=true` to enable snakecase, or leave the default to have it use a subscript.

**Comment by nathanrboyer** [comment]
This isn't working for me on `v0.15.14` or `master`.
```julia
@latexify f_b(x) = x^2
```
![image](https://user-images.githubusercontent.com/65452054/164029290-a3645da8-222d-4de1-af23-32bdae433e41.png)
![image](https://user-images.githubusercontent.com/65452054/164029356-ca5f3464-a035-4c6e-95a9-ad5fb6338065.png)
